instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/db/DbSemanticConventions.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.api.instrumenter.db;
+
+import io.opentelemetry.instrumentation.api.instrumenter.SpanNameExtractor;
+
+public final class DbSemanticConventions {
```

Comment:
I put all DB-specific classes in a subpackage - let's evaluate whether this is a good idea